Osiris/Hooks.cpp

```diff
@@ -193,6 +192,7 @@ static float __stdcall getViewModelFov() noexcept
 static void __stdcall drawModelExecute(void* ctx, void* state, const ModelRenderInfo& info, matrix3x4* customBoneToWorld) noexcept
 {
     if (interfaces.engine->isInGame()) {
+        Visuals::colorWorld();
```

Comment:
You invoke Visuals::colorWorld() every time drawModelExecute hook is called. Call it only once per frame.